types/params.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	maxBlockSizeBytes = 104857600 // 100MB
+	_100MiB = 104857600
```

Comment:
👎  we're losing meaning here

Reply:
The problem though is that a plain maxBlockSizeBytes itself requires having to look it up and isn't documented anywhere, IMHO `_100MiB` is self documenting.

Reply:
I agree, but how about the following: what if somebody else starts using _100MiB and later you decide to change it to _120MiB. you accidentally change both usages and break the code (or worse).

Reply:
I think having to look it up is fine. We shouldn't ruin the variable name :P 

We do need to document this somewhere though.

Reply:
Aye aye, I'll revert the variable.